.github/workflows/wheels.yml

```diff
@@ -14,33 +14,143 @@ on:
 
 jobs:
   build_wheels:
+
     name: Build wheels on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        # os: [ubuntu-latest, windows-latest, macos-latest]
-        os: [ubuntu-latest]
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
 
     steps:
-      - uses: actions/checkout@v2
 
-      - uses: actions/setup-python@v2
-        name: Install Python
+      - name: Acquire sources
+        uses: actions/checkout@v2
+
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        env:
+          # Increase this value to reset cache if requirements*.txt has not changed
+          CACHE_NUMBER: 2
+        with:
+          path: |
+            ~/.cache/pip
+            ~/conda_pkgs_dir
+          key:
+            ${{ runner.os }}-pythonpkg-${{ env.CACHE_NUMBER }}-${{ hashFiles('requirements.txt', 'requirements-docs.txt') }}
+
+      - name: Install Miniconda
+        if: contains(matrix.os, 'windows')
+
+        uses: conda-incubator/setup-miniconda@v2
         with:
-          python-version: '3.7'
+          channels: defaults,conda-forge
+          use-only-tar-bz2: true
 
       - name: Install cibuildwheel
         run: |
-          python -m pip install --upgrade cibuildwheel
+          pip install --upgrade cibuildwheel
 
-      - name: Build wheels
-        run: |
-          python -m cibuildwheel --output-dir wheelhouse
+      # General note:
+      # We wanted to handle this all within a single `cibuildwheel` action,
+      # but things are a bit involved, so we split that between Linux/macOS
+      # 32-bit vs. 64-bit and Windows.
+
+
+      # The workhorse for Linux and macOS 64-bit.
+      #
+      # This uses the vanilla way of `cibuildwheel`. For building on Linux,
+      # it uses the fine `manylinux` images. Because neither `manylinux1`
+      # nor `manylinux2010` apparently ship `blosc-devel` packages, we
+      # switch to `manylinux2014`. For building on macOS, it uses Homebrew.
+
+      - name: Build wheels for Linux and macOS (64-bit)
+        if: contains(matrix.os, 'ubuntu') || contains(matrix.os, 'macos')
+        run: cibuildwheel --output-dir wheelhouse
+        env:
+          CIBW_BUILD: "cp36-manylinux_x86_64 cp37-manylinux_x86_64 cp38-manylinux_x86_64 cp39-manylinux_x86_64 cp36-macosx_x86_64 cp37-macosx_x86_64 cp38-macosx_x86_64 cp39-macosx_x86_64"
+
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+
+          CIBW_ENVIRONMENT: "PYTABLES_NO_EMBEDDED_LIBS=true DISABLE_AVX2=true"
+          CIBW_BEFORE_ALL_LINUX: "yum install -y blosc-devel bzip2-devel hdf5-devel lz4-devel lzo-devel snappy-devel libzstd-devel zlib-devel"
+          CIBW_BEFORE_ALL_MACOS: "brew install c-blosc bzip2 hdf5 lz4 lzo snappy zstd zlib"
+          CIBW_BEFORE_BUILD: "pip install --requirement=requirements.txt --requirement=requirements-dev.txt"
```

Comment:
please add this line to `env`
```
MACOSX_DEPLOYMENT_TARGET: "10.9"
```

Reply:
Thank. But as far as I understand now, this won't help because I used `brew` to install build-time prerequisites, right?

Reply:
Now I got it. The most recent cf3b745 uses
```yaml
env:
    MACOSX_DEPLOYMENT_TARGET: "10.9"
```

instead of
```yaml
env:
    CIBW_ENVIRONMENT_MACOS: "MACOSX_DEPLOYMENT_TARGET=10.9"
```

Thanks!